### ISSUE TITLE
Please ad a Label to the Edit Field and Text to the Run Button in the py-repl Component

### ISSUE THREAD

**Issue by frastlin** [pyscript/pyscript#75]
Hello,
I'm a screen reader user and noticed that the py-repl component is missing text in its button component for run. There is the multiline text field for typing code (that should also have a `<label>` element, then there is an empty `<button></button>` tag with some CSS to show the run command.

1. Visit: [the blog post with an example](https://engineering.anaconda.com/2022/04/welcome-pyscript.html)
2. Select all on the page and copy, then paste into a text editor
3. Find the text: "such as `<py-script>` and `<py-env>`."
4. Observe that between the above text and "Since we already loaded pandas and numpy", there are just blank spaces. This is pretty much how it looks to a screen reader user.

To fix this, please add a label to the multiline input field saying something like: "Python Script Area" and text to the run button saying something like: "Run".

**Comment by fpliger** [comment]
this should have been closed by #322
